vagrant-pretask.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Make user's home directory world-readable
```

Comment:
Do we still need this? We used to need this before we switched to SSHFS for the sync'd folders because NFS needed it. I recently removed this from my own scripts and pulp3 has kept working.

Reply:
Yeah, we still need it for source installs from a shared dir. Something failed without it, I can't remember what.

Reply:
I was using SSHFS, btw